packages/storage-plugin/src/storage.plugin.ts

```diff
@@ -21,10 +22,15 @@ import { tap } from 'rxjs/operators';
 export class NgxsStoragePlugin implements NgxsPlugin {
   constructor(
     @Inject(NGXS_STORAGE_PLUGIN_OPTIONS) private _options: NgxsStoragePluginOptions,
-    @Inject(STORAGE_ENGINE) private _engine: StorageEngine
+    @Inject(STORAGE_ENGINE) private _engine: StorageEngine,
+    @Inject(PLATFORM_ID) private _platformId: any
   ) {}
 
   handle(state: any, event: any, next: NgxsNextPluginFn) {
+    if (isPlatformServer(this._platformId)) {
```

Comment:
@arturovt Does it make sense to modify this check into `if (isPlatformServer(this._platformId) && this._engine === null)`.

In case the user supplies it's own storage engine it will still be used?

Reply:
Sure, good idea.

Reply:
Done